docs/DeveloperGuide.md

```diff
@@ -294,13 +294,15 @@ The `Page` object,
 
 ### 3.6. Storage Component
 
-/* to insert UML diagram */
+![UML diagram for Storage](/diagrams/class/jpeg/Storage_UML_class.jpg)
```

Comment:
Should be diagrams/class/jpeg/Storage_UML_class.jpg

Reply:
It still works actually xD

Reply:
Oh shoot okay, LEMME edit